src/pages/overview/pdf-services-api/howtos/pdf-accessibility-checker-api.md

```diff
@@ -0,0 +1,54 @@
+---
+title: PDF Accessibility Checker | How Tos | PDF Services API | Adobe PDF Services
+---
+<InlineAlert slots="text"/>
+
+PDF Accessibility Checker is currently accessible through the REST API only.
+
+# PDF Accessibility Checker
+
+Accessibility Checker API will check PDF files to see if they meet the machine-verifiable requirements of PDF/UA and WCAG 2.0. It will generate a report that summarizes the findings of the accessibility checks. Additional human remediation may be required to ensure that the reading order of elements is correct and that alternative text tags properly convey the meaning of an image. The report contains links to documentation that assist in manually fixing problems using Adobe Acrobat Pro.
+
+## Input Documents : **Required**
+
+Supported format is PDF (application/pdf). Input PDF with version 1.6 and above is supported.
+
+### Input Document
```

Comment:
you can add this in API param section since here we have only 1 input document and also add 

> Supported format is PDF (application/pdf).

in this only if required.